### PR TITLE
Tooltip for slippage tolerance and transaction deadline added

### PR DESCRIPTION
**Pull request by psalm-ov**
## Description of Changes
This PR added the Tooltip for slippage tolerance and transaction deadline.

_This is for slippage tolerance:_
 `<MouseoverTooltip placement={'top'} text={Description}>`
            &nbsp;&nbsp;`<TooltipContainer>`
              &nbsp;&nbsp;&nbsp;&nbsp;`<QuestionIcon size={16} />`
            &nbsp;&nbsp;`</TooltipContainer>`
          `</MouseoverTooltip>`
image

_This transaction deadline:_
`<span>Transaction deadline: </span>`
          `<MouseoverTooltip placement={'top'} text={Description}>`
            &nbsp;&nbsp;`<TooltipContainer>`
              &nbsp;&nbsp;&nbsp;&nbsp;`<QuestionIcon size={16} />`
            &nbsp;&nbsp;`</TooltipContainer>`
          `</MouseoverTooltip>`
image

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDB-64)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDEV-XXX-description`
* [ ] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [x] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

File: src/components/Tailwind/InputFields/SlippageTolerance.tsx [hunk 1]
```diff
@@ -30,14 +32,22 @@ interface SlippageToleranceProps {
   didChangeValue: (val: string) => void
 }
 
+const TooltipContainer = styled.div`
```
Comment (schystz) [inline src/components/Tailwind/InputFields/SlippageTolerance.tsx]: Pls use tailwindcss, we are deprecating styled components

File: src/components/Tailwind/InputFields/SlippageTolerance.tsx [hunk 2]
```diff
@@ -30,14 +32,22 @@ interface SlippageToleranceProps {
...
+  cursor: pointer;
+`
+
 const SlippageTolerance = ({ value, didChangeValue }: SlippageToleranceProps) => {
   const [customSlippage, setCustomSlippage] = useState('')
-
+  const Description = 'Your transaction will revert if the price changes unfavorably by more than this percentage.'
```
Comment (schystz) [inline src/components/Tailwind/InputFields/SlippageTolerance.tsx]: Pls use translation, e.g. 
```suggestion
  const Description = t('Your transaction will revert if the price changes unfavorably by more than this percentage.')
```

File: src/pages/Tailwind/Swap/modals/SwapSettingsModal.tsx [hunk 3]
```diff
@@ -24,6 +26,10 @@ const SwapSettingsModal = ({
   const dismissGracefully = () => {
     onDismiss()
   }
+  const TooltipContainer = styled.div`
```
Comment (schystz) [inline src/pages/Tailwind/Swap/modals/SwapSettingsModal.tsx]: Pls use tailwind css

**Review by schystz — COMMENTED** [review]
Few comments but otherwise looks good

**Review by tracyarciaga — APPROVED** [review]
LGTM

**Review by schystz — APPROVED** [review]

**Pull request merged**